extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/health/KafkaStreamsHealthCheck.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkus.kafka.streams.runtime.health;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.health.HealthCheck;
+import org.eclipse.microprofile.health.HealthCheckResponse;
+import org.eclipse.microprofile.health.HealthCheckResponseBuilder;
+import org.eclipse.microprofile.health.Readiness;
+import org.jboss.logging.Logger;
+
+import io.quarkus.kafka.streams.runtime.KafkaStreamsTopologyManager;
+
+@Readiness
+@ApplicationScoped
+public class KafkaStreamsHealthCheck implements HealthCheck {
+
+    private static final Logger LOGGER = Logger.getLogger(KafkaStreamsHealthCheck.class.getName());
+
+    @ConfigProperty(name = "quarkus.kafka-streams.topics")
+    public List<String> topics;
+
+    @ConfigProperty(name = "quarkus.kafka-streams.bootstrap-servers")
+    public List<String> bootstrapServers;
+
+    @Inject
+    private KafkaStreamsTopologyManager manager;
+
+    private String commaSeparatedBootstrapServersConfig;
+    private List<String> trimmedTopics;
+
+    @PostConstruct
+    public void init() {
+        commaSeparatedBootstrapServersConfig = String.join(",", (bootstrapServers));
+        trimmedTopics = topics.stream().map(String::trim).collect(Collectors.toList());
+    }
+
+    @Override
+    public HealthCheckResponse call() {
+        HealthCheckResponseBuilder builder = HealthCheckResponse.named("Kafka topic(s) health check").up();
+
+        try {
+            trimmedTopics.forEach(topic -> builder.withData(topic, true));
```

Comment:
I would add a single data with the list of available topic instead of one line with 'true' for each topic